packages/components/src/components/modal/modal.tsx

```diff
@@ -192,14 +193,15 @@ export class Modal {
     if (newValue === true) {
       this.open();
       if (this.allowInjectingStyleToBody) {
+        this.bodyOverflowValue = document.body.style.overflow;
```

Comment:
I store the value in a state. If there is no overflow value is set, it will store at empty string.